components/features/Dashboard/Applications/VideoConference/VideoPlayer/Settings/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from "react";
+import { createPortal } from "react-dom";
+
+const DeviceSettings = () => {
+    return <></>
```

Comment:
If I may ask shad, what will this component contain later?

Reply:
![image](https://user-images.githubusercontent.com/70852167/194755596-117a1c80-85b8-4c5e-af34-1978c10e8c5f.png)
The idea of this component is more or less like this screenshot Alex. In the case where the user have malfunctioning devices, he/she may switch to a new one